grunt/config/browserify.js

```diff
@@ -2,7 +2,7 @@
 
 'use strict';
 
-var envify = require('envify/custom');
+var envify = require('loose-envify/loose-envify');
```

Comment:
Can you leave the `/custom` bit - the API matches envify so we can keep that part the same.